deploy.py

```diff
@@ -22,9 +22,9 @@ def process(xcode, version):
       new_files = list(filter(lambda x : version in x, new_files))
 
   for i in new_files:
-    print 'Unzip file "{}.zip" to {}'.format(i, target)
+    print(f'Unzip file "{i}.zip" to {target}')
     unzip_file(i, target)
-  print '\nUpdate successfully for {}'.format(xcode)
+  print(f'\nUpdate successfully for {xcode}')
```

Comment:
This is a __syntax error__ on all Python versions < v3.6.  Perhaps a more gentle approach would be better.

Reply:
Yeah that makes sense, added a print function from future to handle both python 2 and 3.